### DISCUSSION TITLE
Score result more customizable between numbers or colored bars

### DISCUSSION THREAD

**Discussion by Paul45mec** [ikemen-engine/Ikemen-GO#948] [Feature Request]
### Is your feature request related to a problem? Please describe.

I would like to have the option to customize score result on lifebar to appear like this:

Example : similar to games like Street fighter Alpha series, Crossover Series and SNK games of neogeo from 90's

TIME : 45
LIFE : 100
BONUS : 2000
TOTAL SCORE : 18400

and it would be excellent to have a sound for the number counter.  

Instead of colored bars only representing defense , attack, life etc...

### Describe the solution you'd like

I imagine that it could be a module or a configuration inside the LUA files, but it could be added on the next ikemen go versions.

### Additional context

https://www.youtube.com/watch?v=BaGzkhRo7pg

### Search terms

Score result

**Comment by Amidweiz** [comment]
Is this not something that [score total](https://github.com/ikemen-engine/Ikemen-GO/wiki/Triggers#scoretotal), [Time remaining ÷ Framespercount](https://github.com/ikemen-engine/Ikemen-GO/wiki/Triggers#framespercount), an arbitrary value for a bonus score, and root,life or PlayerID(GetPlayerID(1)),life can't already do. With said vars being displayed at the [end of the match](https://github.com/ikemen-engine/Ikemen-GO/wiki/Triggers/#new_roundtype) with [the text sctrl](https://github.com/ikemen-engine/Ikemen-GO/wiki/State-controllers#text)?

**Reply by Paul45mec** [reply]
Well using this code it always was possible, but i imagine a practical method for all users, something you can customize on lifebar between colored bars or names and numbers for a optional freedom of design.

if you like colored bars or if you like names with number counter.
